www/ui_components/add-server-view.html

```diff
@@ -143,6 +148,10 @@
         opacity: 1.0;
         --iron-icon-fill-color: var(--success-color);
       }
+      /* Reverse key icon */
```

Comment:
I wonder if we could safely reverse **all** icons? Seems like for hamburger, etc., it makes no difference.